features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/ThrottlingPolicies/APIPolicies.js

```diff
@@ -92,36 +108,25 @@ export default class APIPolicies extends Component {
 
         return (
             <div>
-                <AppBar position="static" >
-                    <Toolbar style={{minHeight:'30px'}}>
-                        <IconButton color="contrast" aria-label="Menu">
-                            <MenuIcon />
-                        </IconButton>
-                        <Link to={"/policies/api_policies/create/"}>
-                             <Button color="contrast">Add Policy</Button>
-                        </Link>
-                    </Toolbar>
-                </AppBar>
                 <Message ref={a => this.msg = a}/>
-                <Grid container>
+                <Grid container justify="center" alignItems="center">
                     <Grid item xs={12}>
-                        <Paper>
-                            <Typography className="page-title" type="display1" gutterBottom>
-                               API Rate Limits
+                    
+                        <div className={styles.titleWrapper}>
```

Comment:
This class name will fail in prod build , you need to get class name from classes property